src/Console/Command/SelfUpdate.php

```diff
@@ -0,0 +1,224 @@
+                'Checks what updates are available.'
+            )
+        ;
+    }
+
+    /**
+     * Execute the command.
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->output = $output;
+        $this->version = $this->getApplication()->getVersion();
+
+        /**
+         * Check for ancilliary options
+         */
+        if ($input->getOption('rollback')) {
+            $this->rollback();
+            return;
+        }
+
+        if ($input->getOption('check')) {
+            $this->printAvailableUpdates();
+            return;
+        }
+
+        $this->updateToStableBuild();
+    }
+
+    /**
+     * Perform update using phar-updater configured for stable versions.
+     */
+    protected function updateToStableBuild()
+    {
+        $this->update($this->getStableUpdater());
+    }
+
+    /**
+     * Get phar-updater instance.
+     */
+    protected function getStableUpdater()
+    {
+        $updater = new Updater(null, false);
+        $updater->setStrategy(Updater::STRATEGY_GITHUB);
+        return $this->getGithubReleasesUpdater($updater);
+    }
+
+    /**
+     * Perform in-place update of phar.
+     */
+    protected function update(Updater $updater)
+    {
+        $this->output->writeln('Updating...'.PHP_EOL);
+        try {
+            $result = $updater->update();
+
+            $newVersion = $updater->getNewVersion();
+            $oldVersion = $updater->getOldVersion();
+        
+            if ($result) {
+                $this->output->writeln('<fg=green>PHPMND has been updated.</fg=green>');
+                $this->output->writeln(sprintf(
+                    '<fg=green>Current version is:</fg=green> <options=bold>%s</options=bold>.',
+                    $newVersion
+                ));
+                $this->output->writeln(sprintf(
+                    '<fg=green>Previous version was:</fg=green> <options=bold>%s</options=bold>.',
+                    $oldVersion
+                ));
+            } else {
+                $this->output->writeln('<fg=green>PHPMND is currently up to date.</fg=green>');
+                $this->output->writeln(sprintf(
+                    '<fg=green>Current version is:</fg=green> <options=bold>%s</options=bold>.',
+                    $oldVersion
+                ));
+            }
+        } catch (\Exception $e) {
+            $this->output->writeln(sprintf('Error: <fg=yellow>%s</fg=yellow>', $e->getMessage()));
+        }
+        $this->output->write(PHP_EOL);
+    }
+
+    /**
+     * Attempt to rollback to the previous phar version.
+     */
+    protected function rollback()
+    {
+        $updater = new Updater(null, false);
+        try {
+            $result = $updater->rollback();
+            if ($result) {
+                $this->output->writeln('<fg=green>PHPMND has been rolled back to prior version.</fg=green>');
```

Comment:
is <info> different from <fg=green> ? I would prefer use <danger> , <info>, <warning> and so on